control/controlhttp/http_test.go

```diff
@@ -0,0 +1,389 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package controlhttp
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strconv"
+	"sync"
+	"testing"
+
+	"tailscale.com/net/socks5"
+	"tailscale.com/types/key"
+)
+
+func TestControlHTTP(t *testing.T) {
+	tests := []struct {
+		name  string
+		proxy proxy
+		//rejectHTTP bool
+	}{
+		{
+			name:  "no_proxy",
+			proxy: nil,
+		}, // direct connection
```

Comment:
move this comment above the block like the cases below?

Reply:
Oops, dunno how that happened, must have been in the wrong spot and got gofmt'd more wrong.